### PR TITLE
Added ratings bar and avg star count

### PR DESCRIPTION
**Pull request by zman811**
Added in a bar that shows how many of each star reviews it has

### CODE REVIEW

**Review by hangyin2020 — COMMENTED** [review]
looks good to me

**Review by hangyin2020 — APPROVED** [review]

File: client/src/components/Reviews/components/Ratings.jsx [hunk 1]
```diff
@@ -6,13 +6,52 @@ const Container = styled.div`
...
+const AllRatings = styled.div`
+  text-align: left;
+`;
 
 function Ratings({ reviewsMetadata }) {
-  // TODO parse the data that we get passed in to render things
+  let total = 0;
+  let totalStars = 0;
+  let highest = 0;
+  Object.values(reviewsMetadata.ratings).forEach((num) => { total += num; });
```
Comment (slargman) [inline client/src/components/Reviews/components/Ratings.jsx:26]: When you want to compute a single number from an array, `.reduce()` is usually what you want to reach for:

`Object.values(reviewsMetadata.ratings).reduce((numberOfRating, total) => numberOfRating + total, 0)`

File: client/src/components/Reviews/components/Ratings.jsx [hunk 2]
```diff
@@ -6,13 +6,52 @@ const Container = styled.div`
...
+  const temp = Object.entries(reviewsMetadata.ratings)
+  temp.forEach((item) => { totalStars += item[0] * item[1]; });
```
Comment (slargman) [inline client/src/components/Reviews/components/Ratings.jsx:28]: Same here. You could do `totalStars = Object.entries(...).reduce(...);`

File: client/src/components/Reviews/components/Ratings.jsx [hunk 3]
```diff
@@ -6,13 +6,52 @@ const Container = styled.div`
...
+  const avgStars = totalStars / total;
+  Object.values(reviewsMetadata.ratings).forEach((num) => {
```
Comment (slargman) [inline client/src/components/Reviews/components/Ratings.jsx:30]: `Math.max(...Object.values())`

**Review by slargman — APPROVED** [review]
Looks ready to merge. I had some suggestions for simplifications you could make, but the functionality is good.

**Pull request merged**